### PR TITLE
New chart landing page

### PR DESCRIPTION
**Pull request by haotran-avaiga**
I have some issues when working on this ticket:
- we just have the document for some charts.
- I don't know how to put the path of each chart in a tag
- I have to use HTML to style the list of charts, I am not sure it is the best way
- I know we generate the table of properties, but I don't know how to put it into the MD file.

I appreciate it if you put your comments here so I can fix them.

### CODE REVIEW

**Review by FredLL-Avaiga — COMMENTED** [review]
any dark them images Or am I mistaken ?

**Comment by haotran-avaiga** [comment]
> any dark them images Or am I mistaken ?

sorry no, all images are taken from the design file, so there is no dark theme in image. Fabien told me we will use Taipy to generate the images (both light and dark versions)

File: gui/doc/chart.md [hunk 1]
```diff
@@ -3,30 +3,268 @@ Displays data sets in a chart or a group of charts.
 The chart control is based on the [plotly.js](https://plotly.com/javascript/)
 graphs library.
 
-A chart control can hold several traces, that can display individual data sets.
+Plotly is an amazing graphing library that provides a very large number of visual
```
Comment (FredLL-Avaiga) [inline gui/doc/chart.md]: Amazing ?

File: gui/doc/chart.md [hunk 2]
```diff
@@ -3,30 +3,268 @@ Displays data sets in a chart or a group of charts.
...
+    value1,
+    value2,
+    value3
+]
+    
+page = "<|...|chart|...|index_property={values}|...|>"
+```
+is equivalent to
+```
+page = "<|...|chart|...|index_property[1]={value1}|index_property[2]={value2}|index_property[3]={value3}|...|>"
```
Comment (FredLL-Avaiga) [inline gui/doc/chart.md:234]: <|...|chart|...|index_property={value1}|index_property[1]={value1}|index_property[2]={value2}|index_property[3]={value3}|...|>

**Review by FredLL-Avaiga — APPROVED** [review]

File: gui/doc/chart.md (hunk 2 above)
Comment (FabienLelaquais) [reply gui/doc/chart.md:234]: I'm sorry. I don't get your point...
(except that 'index_property' should be 'indexed_property' everywhere, which is too long)

File: gui/doc/chart.md (hunk 1 above)
Comment (FabienLelaquais) [reply gui/doc/chart.md]: Hum. A bit too much?
(that's what I think, nevertheless)

Comment (FabienLelaquais) [reply gui/doc/chart.md]: Would  'impressive' do? 'Comprehensive'?

Comment (FredLL-Avaiga) [reply gui/doc/chart.md]: Comprehensive I like

File: gui/doc/chart.md (hunk 2 above)
Comment (FredLL-Avaiga) [reply gui/doc/chart.md:234]: when you specify an array, the first value of the array becomes the default value.
ie if your array is shorter than the number of traces, the extra traces inherit from the 1st value of the array (like they would if the non index property was not an array)

File: gui/doc/chart.md (hunk 1 above)
Comment (FabienLelaquais) [reply gui/doc/chart.md]: 'vast' it is...

File: gui/doc/chart.md (hunk 2 above)
Comment (FabienLelaquais) [reply gui/doc/chart.md:234]: Yes.
So I guess your
`index_property={value1}|index_property[1]={value1}`
was meant to be
`index_property={value1}`

Comment (FredLL-Avaiga) [reply gui/doc/chart.md:234]: Which is the same in the end (and might be clearer ?)

**Pull request merged**